proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+  </param>
+  <param name="coolingLevelAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="horizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="verticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="frontVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backTiltAngleAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="headSupportHorizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="headSupportVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageEnabledAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageModeAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageCushionFirmnessAvailable" type="Boolean" array="false">
+  </param>
+  <param name="memoryAvailable" type="Boolean" array="false">
+  </param> 
+</struct> 
+
+<struct name="RemoteControlCapabilities">
+  :
+  :
+  <param name="seatControlCapabilities" type="SeatControlCapabilities" mandatory="false" minsize="1" maxsize="100" array="true">
+    <description>
+      If included, the platform supports seat controls.
+    </description >
+  </param>
+</struct>
+
+<struct name="ModuleData">
+  :
+  :
+  <param name="seatControlData" type="SeatControlData" mandatory="false">
+  </param>
+</struct>
+```
+#### Additions to HMI_API
+The changes are the same as mobile api changes, they are not duplicated here. 
+
+## Potential downsides
+
+Due to the lack of an agreed control resource ID/Zone scheme, we define `SupportedSeat` enum as a work around to allow control both driver seat and front passenger seat.
+We shall have a global remote-control-resource ID/Zone scheme, that assigns an ID to each remote controllable resource within a vehicle.
+
+## Impact on existing code
+
+The proposal has the impact on:
+
+RPC changes:
+
+- Add new `SEAT` module and corresponding capabilities and control data types to RC APIs.
+- The proposal has the impact on: `SetInteriorVehicleData`, `GetInteriorVehicleData`, `OnInteriorVehicleData`, `GetSystemCapability`
+
+HMI changes:
+
+- New HMI API parameters support
+- SDL HMI shall support the emulation of new RC modules.
+
+Mobile iOS/Android SDK changes:
+
+- New Mobile API parameters support
+
+SDL core changes:
+
+- New API parameters support
+- New remote control modules support
+
+Mobile Apps:
+
+- There is no impact on existing mobile applications or SDL integrations if they do not used the new module or new parameters.
+
+Policy:
+
+- Policies need to support new ModuleTypes (`SEAT`)
+- SDL need to transfer RPCs with new parameters to appropriate vehicle's module. The response resultCode depends on vehicle's result of processing.
```

Comment:
Please update to "SDL needs to transfer RPCs with new parameters to appropriate vehicle modules."

Reply:
changed, thank you